test/hu/openig/screen/items/PlanetScreenTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2008-2017, David Karnok
+ * The file is part of the Open Imperium Galactica project.
+ *
+ * The code should be distributed under the LGPL license.
+ * See http://www.gnu.org/licenses/lgpl.html for details.
+ */
+package hu.openig.screen.items;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.awt.Rectangle;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Planet screen unit test.
+ *
+ * @author rdubisz
+ * @since 0.95.210
+ */
+public class PlanetScreenTest {
```

Comment:
Unit test makes no sense for Open-IG and I've omitted them.

Reply:
Ok, I will remove it. POM was also added just to tell IDE where it can find tests sources, not to bundle the application. It's your project so I will do as you say :) Just my personal experience is that tests makes development much easier even in GUI application.